semgrep-core/src/parsing/tree_sitter/Parse_swift_tree_sitter.ml

```diff
@@ -1426,9 +1428,11 @@ and map_expr_hack_at_ternary_call_suffix (env : env)
     (x : CST.expr_hack_at_ternary_call_suffix) =
   map_value_arguments env x
 
-and map_expression (env : env) (x : CST.expression) =
+and map_expression (env : env) (x : CST.expression) : G.expr =
   match x with
-  | `Simple_id x -> map_simple_identifier env x
+  | `Simple_id x ->
+      let id = map_simple_identifier env x in
+      G.N (G.Id (id, G.empty_id_info ())) |> G.e
```

Comment:
You can use H2.name_of_id
where module H2 = AST_generic_helpers (H is usually used for Parse_tree_sitter_helpers.ml)

Reply:
Addressed in #4912